api/errors/builder.go

```diff
@@ -0,0 +1,93 @@
+package errors
+
+// This file contains all of the errors that can be generated from the
+// docker/builder component.
+
+import (
+	"net/http"
+
+	"github.com/docker/distribution/registry/api/errcode"
+)
+
+var (
+	// ErrorCodeAtLeastOneArg is generated when the parser comes across a
+	// Dockerfile command that doesn't have any args.
+	ErrorCodeAtLeastOneArg = errcode.Register(errGroup, errcode.ErrorDescriptor{
```

Comment:
The primary goal of the API package is consistent error reporting and json error format. I18N is really a UI issue. We should be considering this use case, but not overly optimizing the use of `errcode` for i18n.

This is why we allowed different messages in the original `errcode`. Otherwise, the mapping of codes and actions becomes influenced by the UI rather than sane error handling.